DDDEastAnglia.Tests/Services/Messengers/Email/Templates/SessionCreated.txt

```diff
@@ -0,0 +1,2 @@
+﻿Title: [SessionTitle]
```

Comment:
This file should be deleted: the email template are in the Emails.resx file now.